packages/demo/scss/index.scss

```diff
@@ -23,7 +23,7 @@
     gap: 0.5rem 0.75rem;
     align-items: baseline;
     margin-bottom: 1.5rem;
-    font-size: 1.125rem; /* Based on the `heading-small-text` style. */
+    font-size: 1.25rem; /* Based on the `heading-4-bold` style. */
```

Comment:
Shouldn't it be a different heading style then so the font size remains unchanged?

Reply:
There is no style with the same values. But this is only on mobile 🤷